source/mobile/mobile-faq.rst

```diff
@@ -113,6 +112,74 @@ The following options are available for securing your push notification service:
 
 .. Note:: For configuration details, see guides for :doc:`deploying the Mattermost App Store and Google Play apps <mobile-appstore-install>` and :doc:`deploying your own version of the apps <mobile-compile-yourself>`.
 
+How do I deploy Mattermost with Enterprise Mobility Management providers?
+--------------------------------------------------------------------------
+
+Mattermost enables customers with high privacy and custom security requirements to deploy mobile applications and push notification services using keys that they alone control.
+
+:doc:`Learn more about using AppConfig for EMM providers <mobile-appconfig>`.
+
+How do I host the Mattermost push notification service?
+----------------------------------------------------------
+
+First, you can use the :doc:`Mattermost Hosted Push Notification Service (HPNS) <mobile-hpns>`. Organizations can also `host their own push proxy server instead <https://developers.mattermost.com/contribute/mobile/push-notifications/service/>`_. This is applicable when you want to
+
+1. customize the `Mattermost mobile apps <https://github.com/mattermost/mattermost-mobile>`_,
```

Comment:
maybe best this link https://developers.mattermost.com/contribute/mobile/build-your-own/ ? 0/5